### ISSUE TITLE
Linux Container - Running SQL script file using sqlcmd in a shell script fails

### ISSUE THREAD

**Issue by iqans** [Microsoft/mssql-docker#308]
I've been trying to run a SQL script on build time of container. I have a shell script that looks something like this:
`echo "Starting SqlServr"
/opt/mssql/bin/sqlservr &
sleep 30 | echo "Waiting for 30s to start Sql Server"
echo "Restoring DB."
/opt/mssql-tools/bin/sqlcmd -U sa -P <valid-password> -i RestoreDb.sql
echo "DB restored."
echo "Deleting backup files."
rm -rf /work/*.bak`

I run this from Dockerfile using: `RUN`

I'm getting error `': Invalid filename.ql`

When I run the same command from shell `/opt/mssql-tools/bin/sqlcmd -U sa -P <valid-password> -i RestoreDb.sql` it works perfectly fine.
Also when I replace `-i RestoreDb.sql` to `-Q <file-content>` it also works perfectly fine.

Looks like something to do with `sqlcmd` and `bash`.

**Comment by twright-msft** [comment]
Can you please share your Dockerfile, .sh script, and .sql script (redacting anything sensitive)?

**Comment by iqans** [comment]
Here's the repo: [mssql-linux-container-test](https://github.com/iqans/mssql-linux-container-test)

I've tried again making some changes.
It worked on Linux container on Linux machine.
But did not work on Linux container on Windows machine.

Also there are chances of inconsistent line ending. I've seen `\r` at the end of line in shell script when I edit on windows machine. This maybe the issue.

**Comment by iqans** [comment]
![image](https://user-images.githubusercontent.com/10431508/40050883-b43ef66c-5830-11e8-95d6-f39da326ec7e.png)

**Comment by fabiopucci** [comment]
Check EOL character. It needs to be set to LF only since container runs linux.

**Comment by iqans** [comment]
yes. did it. But, this will be the issue when created scripts on windows dev machine. 
And main problem is with the error message, it doesn't provide correct information on what went wrong.

**Comment by luigilizcano1003** [comment]
> Check EOL character. It needs to be set to LF only since container runs linux.

Thanks for your help. @iqan

**Comment by tracker1** [comment]
@iqan what are you using to generate the scripts... if nothing else, have a utility to change from whatever you're saving into the expected format... if it's UTF-8/16, stripping \r's etc.
